index.html

```diff
@@ -28,9 +28,40 @@ <h2 class="mk-heading mk-heading--xl mk-m-border">Why Metal<sup>3</sup></h2>
          backend for Machine objects from the Cluster API. <br> These components integrate
          seamlessly to leverage the Kubernetes ecosystem and automate the provisioning
          and management of bare-metal infrastructure.
-       </p>
+       </p><br>
+       <p>
+        Metal<sup>3</sup> mission is to empower organizations with a flexible, 
```

Comment:
let's move this up as first statement
also use "The Metal3 project" instead of just Metal3